README.md

```diff
@@ -42,6 +42,28 @@ $ cp .env.example .env
 ```
 $ yarn
 ```
+### Additional requisites for Installation on a Windows device
+
+1. Run Windows Powershell as administrator and type in the following command:
+
+```
+$ wsl --install
+```
+2. Download Ubuntu from Microsoft store or type in the following command:
+
+```
+$ wsl --install -d ubuntu
+```
+
+3. Run ubuntu on your Windows computer. Afterwards, it will ask you to create initial username and password.
+
+4. Run the following command after you registered into Ubuntu
+
+```
+$ sudo apt update
+```
+
+5. start your computer to allow these installs to take effect.
```

Comment:
Let's not specify the steps to install WSL. These can change at any time and we're better off directing future readers to the official WSL docs :). 

Could you please add a note somewhere suitable that Windows isn't natively supported, and to recommend them to install and use WSL?

Reply:
Will add a note into that will specify Windows is not natively supported :)